src/stores/Widgets/SpectralProfileWidget/SpectralProfileWidgetStore.ts

```diff
@@ -83,11 +83,22 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
     readonly profileSelectionStore: SpectralProfileSelectionStore;
     readonly fittingStore: ProfileFittingStore;
 
+    /**
+     * Set region for generating moment maps.
+     *
+     * @param fileId - File ID number.
+     * @param regionId - Region ID number.
+     */
     @override setRegionId = (fileId: number, regionId: number) => {
         this.regionIdMap.set(fileId, regionId);
         this.clearXYBounds();
     };
 
+    /**
+     * Set spectral coordinates for moment maps.
+     *
+     * @param coordStr - A string of <i>Spectral type (unit)</i> like Frequency (GHz).
+     */
     @action setSpectralCoordinate = (coordStr: string) => {
```

Comment:
I think the moment map coordinate setting is synchronized to the profiler, so this function is mainly for the profiler.